documentation/htc_workloads/managing_data/data-transfer.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Data Transfer to Jobs - OSG Guide"
+output: html_document
+---
+# Data Staging and Transfer to Jobs¶
+
+Purpose
+This guide introduces the three 
+
+
+# Overview¶
+
+OSG-managed Access Points have two locations for uploading files (data and software) that are needed for running jobs, `/home` and `/protected`. Where you store your files and how your files are made accessible to your jobs depends on how much data is needed or produced by your jobs.
+
+In general, users are responsible for managing data in these folders and for using appropriate mechanisms for delivering data to/from jobs, as detailed below. Each is controlled with a quota and should be treated as temporary storage for active job execution. OSG Connect has no routine backup of data in these locations, and users should remove old data after jobs complete, in part, to make room for future submissions. 
+
+
+# Data Locations
+
+# Check Your Quota and Available Space
+
+Users have the following default quotas: 
+
+/home 50 GB
+/protected 500 GB
+
+There are two primary ways to check your remaining disk space: 
+
+## Option 1: Quota Usage Displayed Upon Login
+
+Your quota status will be automatically displayed when you login:
+
+Disk utilization for username:
+/public   : [                        ] 0% (0/500000 MB)
+/home     : [ #                      ] 4% (2147/53687 MB)
```

Comment:
```suggestion
```
Disk utilization for username:
/public   : [                        ] 0% (0/500000 MB)
/home     : [ #                      ] 4% (2147/53687 MB)
```
```